circle.yml

```diff
@@ -27,7 +27,7 @@ deployment:
   #  commands:
   #    - ./gradlew -i artifactoryPublish
   release:
-    tag: /[0-9]+(\.[0-9]+)+/
+    tag: /[0-9]+(\.[0-9]+)+(-rc[0-9])?/
```

Comment:
The former is intentional, the latter is just to do the minimal thing that will let me make progress. Happy to use something more extensive like `/[0-9]+(\.[0-9]+)+(-rc[0-9]+)?(-alpha[0-9]+)?/`.